system/src/system_task.cpp

```diff
@@ -794,6 +802,52 @@ void* system_internal(int item, void* reserved)
     case 3: {
         return reinterpret_cast<void*>(system_cloud_get_socket_handle());
     }
+#if PLATFORM_ID != PLATFORM_GCC && PLATFORM_ID != PLATFORM_NEWHAL
```

Comment:
This should be removed from normal release builds. If we need some of these for testing, let's wrap them under some normally-disabled compile-time macro, otherwise add a set of unstable APIs